admin/js/html5-fallback.js

```diff
@@ -0,0 +1,45 @@
+( function( $ ) {
```

Comment:
Why do we need this fallback JS?

Reply:
The idea is to give a similar experience to users when they use a browser that doesn't support html5 types, when data and time fields degrade to simple text.